beacon-chain/attestation/service_test.go

```diff
@@ -25,8 +25,17 @@ func TestUpdateLatestAttestation_UpdatesLatest(t *testing.T) {
 	defer internal.TeardownDB(t, beaconDB)
 	ctx := context.Background()
 
+	var validators []*pb.Validator
+	for i := 0; i < 64; i++ {
+		validators = append(validators, &pb.Validator{
+			Pubkey:          []byte{'A'},
```

Comment:
is it ok that all 64 validators have the same pub key?
 just to make sure it doesn't affect the test integrety

Reply:
I agree to use different ones - just in case this doesn't catch regression in future changes

Reply:
yeah that was an oversight, will change it